### PR TITLE
Fix coq-mathcomp-multinomials.dev according to math-comp/multinomials#45

### PR DESCRIPTION
**Pull request by pi8027**
Currently, .vo files are missing in the installation.

### CODE REVIEW

File: extra-dev/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.dev/opam [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-name: "coq-mathcomp-multinomials"
 maintainer: "pierre-yves@strub.nu"
 homepage: "https://github.com/math-comp/multinomials"
 bug-reports: "https://github.com/math-comp/multinomials/issues"
 dev-repo: "git+https://github.com/math-comp/multinomials.git"
 license: "CeCILL-B"
 authors: ["Pierre-Yves Strub"]
-build: [ "dune" "build" "-p" name "-j" jobs ]
+build: [
+  [ "bash" "./configure" ]
```
Comment (palmskog) [inline extra-dev/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.dev/opam:9]:
```suggestion
  [ "./configure" ]
```
Does this really have to be bash? Any shell should do for the invocation, right?

Comment (pi8027) [reply extra-dev/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.dev/opam:9]: I think your suggestion makes sense but I just copy-pasted it from math-comp/multinomials#45. cc: @erikmd

**Comment by pi8027** [comment]
BTW, shouldn't we move `coq-mathcomp-multinomials.1.5.1` from extra-dev to released?

File: extra-dev/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.dev/opam (hunk 1 above)
Comment (pi8027) [reply extra-dev/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.dev/opam:9]: Let's remove it and see how it goes anyway.

**Pull request merged**